src/nupic/regions/TMRegion.cpp

```diff
@@ -251,12 +252,13 @@ void TMRegion::compute() {
     auto predictive = tm_->getPredictiveCells(); // sparse
     if (args_.orColumnOutputs) {
       // aggregate to columns
-      active = VectorHelpers::sparse_cellsToColumns<UInt>(active, args_.cellsPerColumn);
-      predictive = VectorHelpers::sparse_cellsToColumns<UInt>(predictive, args_.cellsPerColumn);
+      active = VectorHelpers::sparse_cellsToColumns<CellIdx>(active, args_.cellsPerColumn);
+      predictive = VectorHelpers::sparse_cellsToColumns<CellIdx>(predictive, args_.cellsPerColumn);
     }
     SDR& sdr = out->getData().getSDR();
-    VectorHelpers::unionOfVectors<UInt>(sdr.getSparse(), active, predictive);
-    sdr.setSparse(sdr.getSparse()); // to update the cache in SDR.
+    vector<CellIdx> uni;
+    VectorHelpers::unionOfVectors<CellIdx>(uni, active, predictive);
```

Comment:
Before this was an in-place update and now it makes a new vector and swaps it in...

Reply:
Good spot! 
This was for mismatch in SDR_sparse_t, and vector<CellIdx> //you can change CellIdx; 
but got fixed later by binding connections::CellIdx and sdr::ElemSparse

TL;DR reverted back